AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1051,6 +1051,7 @@ sprashivalka.com##div[class="a1 "][style="height: 90px;"]
 !*
 serfery.ru##.b-footer-callout
 chip.com.tr#?#.altbolum:has(> div > div > .newslettergroup)
+svoboda.org##a[href$="/subscribe.html"]
```

Comment:
we shouldn't block such elements

![Screenshot 2019-07-09 at 19 15 00](https://user-images.githubusercontent.com/45171506/60905500-e3ebbd80-a27d-11e9-81c0-83a2e4aa81c8.png)